app/pb_kits/playbook/pb_button/_button.jsx

```diff
@@ -16,105 +16,104 @@ type ButtonPropTypes = {
   icon?: String,
   loading?: Boolean,
   newWindow?: Boolean,
-  size: 'large' | 'medium' | 'small',
+  size: "large" | "medium" | "small",
   text?: String,
-  type: 'inline' | null,
-  htmlType: String | 'button',
+  type: "inline" | null,
+  htmlType: String | "button",
   value?: String | null,
-  variant: 'primary' | 'secondary' | 'link',
-  wrapperClass: String,
-}
+  variant: "primary" | "secondary" | "link",
+  wrapperClass: String
```

Comment:
Wondering where some of the formatting changes came from here. For instance, this one should end with a comma `,` according to eslint rules. Also, semicolons are a thing of the past really AFAIK unless vanilla js.

Reply:
hum... looks like my Prettier plugin is not configured properly

Reply:
Not a huge deal really. I tend to fix these sort of things as I move through enhancements.